src/Server_Command.php

```diff
@@ -80,11 +80,12 @@ function __invoke( $_, $assoc_args ) {
 
 		// Get the path to the router file
 		$command_root = WP_CLI\Utils\phar_safe_path( dirname( __DIR__ ) );
```

Comment:
Use an import for `WP_CLI\Utils`:
```suggestion
		$command_root = Utils\phar_safe_path( dirname( __DIR__ ) );
```